src/_P096_eInk.ino

```diff
@@ -0,0 +1,603 @@
+            }          
+            else if(subcommand.equalsIgnoreCase(F("c")) && argCount == 4)
+            {
+              eInkScreen->drawCircle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), Plugin_096_ParseColor(sParams[3]));
+            }          
+            else if(subcommand.equalsIgnoreCase(F("cf")) && argCount == 5)
+            {
+              eInkScreen->fillCircle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), Plugin_096_ParseColor(sParams[4]));
+              eInkScreen->drawCircle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), Plugin_096_ParseColor(sParams[3]));
+            }
+            else if(subcommand.equalsIgnoreCase(F("t")) && argCount == 7)
+            {
+              eInkScreen->drawTriangle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), sParams[5].toInt(), Plugin_096_ParseColor(sParams[6]));
+            }           
+            else if(subcommand.equalsIgnoreCase(F("tf")) && argCount == 8)
+            {
+              eInkScreen->fillTriangle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), sParams[5].toInt(), Plugin_096_ParseColor(sParams[7]));
+              eInkScreen->drawTriangle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), sParams[5].toInt(), Plugin_096_ParseColor(sParams[6]));
+            }           
+            else if(subcommand.equalsIgnoreCase(F("rr")) && argCount == 6)
+            {
+              eInkScreen->drawRoundRect(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), Plugin_096_ParseColor(sParams[5]));
+            }          
+            else if(subcommand.equalsIgnoreCase(F("rrf")) && argCount == 7)
+            {
+              eInkScreen->fillRoundRect(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), Plugin_096_ParseColor(sParams[6]));
+              eInkScreen->drawRoundRect(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), Plugin_096_ParseColor(sParams[5]));
+            } 
+            else if(subcommand.equalsIgnoreCase(F("px")) && argCount == 3)
+            {
+              eInkScreen->drawPixel(sParams[0].toInt(), sParams[1].toInt(), Plugin_096_ParseColor(sParams[2]));
+            } 
+            else 
+            {
+              success = false;
+            }
+          }
+                    
+        }
+        else
+        {
+          //invalid arguments
+          success = false;
+        }
+
+
+        if(!success)
+        {
+          if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+            addLog(LOG_LEVEL_INFO, F("Fail to parse command correctly; please check API documentation"));
```

Comment:
This will give lots and lots of log entries, if commands are given for other tasks/plugins.

`success = false` is something that will happen in almost all calls to `PLUGIN_WRITE`  (except  for the task the command was meant to be sent)

Reply:
ok you're right. I've removed those log lines.